src/Run/Setup.cs

```diff
@@ -12,7 +12,8 @@ namespace Microsoft.DotNet.Execute
 {
     public class Setup
     {
-        private static readonly string runQuietReservedKeyword = "RunQuiet";
+        private const string runQuietReservedKeyword = "RunQuiet";
+        private const string runToolSettingValueTypeReservedKeyword = "runToolSetting";
```

Comment:
These should be PascalCased. See (12) https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md. https://msdn.microsoft.com/en-us/library/x2dbyw72(v=vs.71).aspx